src/Sonarr.Api.V3/Blocklist/BlocklistController.cs

```diff
@@ -23,9 +23,9 @@ public class BlocklistController : Controller
 
         [HttpGet]
         [Produces("application/json")]
-        public PagingResource<BlocklistResource> GetBlocklist()
+        public PagingResource<BlocklistResource> GetBlocklist([FromQuery] PagingRequestResource paging)
         {
-            var pagingResource = Request.ReadPagingResourceFromRequest<BlocklistResource>();
+            var pagingResource = new PagingResource<BlocklistResource>(paging);
```

Comment:
This drops support for `FilterKey` and `FilterValue` as of right now params for filtering, but we could probably add that behaviour back. This was originally added for backwards compatibility with v2 API.

Reply:
I think this is likely fine with the v4 bump